rendered/manifests/agent-only/daemonset.yaml

```diff
@@ -79,6 +79,14 @@ spec:
           protocol: TCP
         image: quay.io/signalfx/splunk-otel-collector:0.67.0
         imagePullPolicy: IfNotPresent
+        securityContext:
+          allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: true
```

Comment:
Do you know offhand if this will this prevent writing to any non-volume directory?

Reply:
Yes, testing showed that it prevents write operation also to non-volume directories

Reply:
This would be a breaking change for the smart agent receiver and collectd-based monitors